.github/workflows/flamegraph_reporter.yaml

```diff
@@ -0,0 +1,209 @@
+name: Daily Flamegraph Reporter
```

Comment:
I collapsed all daily reports into https://github.com/lambdaclass/ethrex/blob/main/.github/workflows/daily_reports.yaml

Reply:
Thanks! Not sure if it's daily though, I forgot I had named it like that but I think it's supposed to run in each merge to master; I will ask Javi/Ivan.